src/main/java/seedu/address/ui/ConfirmWindow.java

```diff
@@ -103,12 +102,16 @@ public void focus() {
      * Returns confirmation.
      */
     @FXML
-    private void confirm() {
+    private void confirm() throws IOException {
         if (isMeetingClear) {
             logic.clearMeetingBook();
+            logic.getStorage().saveMeetingBook(logic.getModel().getMeetingBook());
+
         } else {
             logic.clearAddressBook();
             logic.clearMeetingBook();
+            logic.getStorage().saveAddressBook(logic.getModel().getAddressBook());
+            logic.getStorage().saveMeetingBook(logic.getModel().getMeetingBook());
```

Comment:
Thanks for fixing the bug! Just a suggestion: Don't you think it would be better design if we moved the storage part to the `logic.clearXXXBook()` method? 

```java
    public void clearAddressBook() {
        requireNonNull(model);
        model.setAddressBook(new AddressBook());
        storage.saveAddressBook(model.getAddressBook());
    }
```

Reply:
suggestion taken